### DISCUSSION TITLE
Setting Brevo SMTP on Supabase

### DISCUSSION THREAD

**Discussion by joaoguidev** [supabase/supabase#22030] [Questions] [answered]
Hello.

Im trying to set Brevo transactional emails to be used by Supabase to send my confimation emails, magic link and so on instead of the built-in Supabase one but, without success.

![image](https://github.com/supabase/supabase/assets/63625334/133f4b20-6661-460d-874f-29aac66ae215)

Brevo gives me the following info to be used on SMTP:

![image](https://github.com/supabase/supabase/assets/63625334/04a0225b-d57d-4836-95fc-8b3d831a5a89)

From the image above, I'm setting Supabase on the studio like so 

- (Brevo) SMTP Server --> (Supabase) Host ** this is: smtp-relay.brevo.com
- (Brevo) Port --> (Supabase) Port number
- (Brevo) Login--> (Supabase) Username *** this one is curious because in Brevo, the Login value is an email address
- (Brevo) I generate a SMTP key --> (Supabase) Password

In Brevo I registered my domain and also registered the sender with an email of the refered domain. I the mentioned email on the Supabase Studio SMTP fields "Sender email".

Has anyone had success implementing this? What am I missing?

Thank you.

**Comment by GaryAustin1** [comment, marked as answer]
![image](https://github.com/supabase/supabase/assets/54564956/1f8ee4d5-757a-4da9-ba1d-f84515fb4a34)
I use my Brevo login email.

**Reply by joaoguidev** [reply]
Hello GaryAustin1.

On the field password on Supabase, you create a SMTP key on Brevo and use it there right?

**Reply by GaryAustin1** [reply]
Yes.
![image](https://github.com/supabase/supabase/assets/54564956/9720b270-75e5-4200-b7de-7aaadce64f21)

**Reply by joaoguidev** [reply]
It is strange. On the Supabase auth log I get "no such host" for smtp-relay.brevo.com.

![image](https://github.com/supabase/supabase/assets/63625334/f57e0f95-7460-4c0c-9e26-902dfac05860)

**Reply by joaoguidev** [reply]
OMG!!!
There was a space before smtp-relay.brevo.com.
Forget it.

It worked.

**Comment by someshfengde** [comment]
it's not working for me there's no log for from supabase side when I try to send magic link it's getting sent sucessfully but on recipient's mail address it's not receiving

**Reply by khalilonline99** [reply]
Shouldn't you check the log at Brevo? Whats the error message there?

**Reply by someshfengde** [reply]
I've tried to figure it out for like 2 hours then just shifted to resend. They have great integration with supabase setup took me just 1 minute.

**Comment by GreeningSiren** [comment]
Hello,
I'm getting error "535 5.7.8 Authentication failed" even though my smtp key is correct

**Reply by Adityahulk** [reply]
Hey @GreeningSiren were you able to resolve this ?, I am facing the same issue.

**Reply by narongdejsrn** [reply]
Facing the same issue too

**Comment by Izalfront** [comment]
excuse me, how do I connect supabase to brevo? for email verification

**Reply by GaryAustin1** [reply]
You need to follow their SMTP instructions and fill in the fields as discussed above.

**Comment by asaphcyusa01** [comment]
@GaryAustin1 I did all that but still have this error message: {"auth_event":{"action":"user_invited","actor_id":"00000000-0000-0000-0000-000000000000","actor_username":"supabase_admin","actor_via_sso":false,"log_type":"team","traits":{"user_email":"cyusaasaph@gmail.com","user_id":"f8408d90-c616-44d7-a393-b1b5f95a9182"}},"component":"api","error":"535 5.7.8 Authentication failed","level":"error","method":"POST","msg":"500: Error sending invite email","path":"/invite","referer":"immerserwanda.com","remote_addr":"34.247.6.71","request_id":"92ecd276849cbddc-DUB","time":"2025-04-11T19:17:06Z"}

**Reply by GaryAustin1** [reply]
535 error usually is bad password/credentials to the SMTP provider.

**Comment by TrimeraDev** [comment]
I also had the issue with the space in the host. (supabase really should trim that ...)
And for the Username i want to make clear that it's not the email you used to sign up to brevo. Where you generate your SMTP keys in Brevo they have information about the port number, SMPT-Server AND a Login email: {randomString}@smtp-brevo.com.


thanks for the thread. it helped me : )

**Comment by And-0** [comment]
I had the same issue just now but managed to fix it thanks to something @TrimeraDev said. My issue was that on the Brevo SMTP&API page, the login it gave me was actually just my email address I used to sign up to Brevo, _not_ a login in the form of {randomString}@smtp-brevo.com. No idea why. Once I regenerated the SMTP Login and Master password, I got a correct login that worked immediately with Supabase.
